libraries/AP_Scripting/lua/src/linit.c

```diff
@@ -48,7 +48,7 @@ static const luaL_Reg loadedlibs[] = {
 //  {LUA_OSLIBNAME, luaopen_os},
   {LUA_STRLIBNAME, luaopen_string},
   {LUA_MATHLIBNAME, luaopen_math},
-  {LUA_UTF8LIBNAME, luaopen_utf8},
+//  {LUA_UTF8LIBNAME, luaopen_utf8},
```

Comment:
please use !defined(ARDUPILOT_BUILD) to make life a bit easier when we update lua